hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerDoubleBuffer.java

```diff
@@ -86,10 +89,10 @@
 
   private final boolean isRatisEnabled;
 
-  public OzoneManagerDoubleBuffer(OMMetadataManager omMetadataManager,
-      OzoneManagerRatisSnapshot ozoneManagerRatisSnapShot) {
-    this(omMetadataManager, ozoneManagerRatisSnapShot, true);
-  }
+  /**
+   * function which will get term associated with the transaction index.
+   */
+  private Function<Long, Long> termGet = null;
 
   public OzoneManagerDoubleBuffer(OMMetadataManager omMetadataManager,
       OzoneManagerRatisSnapshot ozoneManagerRatisSnapShot,
```

Comment:
For this constructor, termGet will be null. But a daemon is initialized to flushTransactions. Could it lead to NullPointer?

Reply:
This constructor is removed, as now we use Builder. And also for non-HA this termGet is not used, so we don't need this to be set. So, skipped setting this in OzoneManagerProtocolServerSideTranslatorPB. (Previously this constructor used to be called from this class)